packages/components/src/components/post-footer/post-footer.tsx

```diff
@@ -0,0 +1,120 @@
+import { Component, Element, h, Host, Prop, State } from '@stencil/core';
+import { version } from '@root/package.json';
+import { breakpoint } from '../../utils/breakpoints';
+
+/**
+ * @slot grid-{1|2|3|4}-title - Slot for the accordion headers (mobile).
+ * @slot grid-{1|2|3|4} - Slot for the accordion bodies (mobile) and the grid cells (tablet, desktop).
+ * @slot socialmedia - Slot for the social media links.
+ * @slot app - Slot for the app links.
+ * @slot businesssectors - Slot for the business sectors links.
+ * @slot meta - Slot for the meta links.
+ * @slot copyright - Slot for the copyright text.
+ */
+@Component({
+  tag: 'post-footer',
+  shadow: true,
+  styleUrl: './post-footer.scss',
+})
+export class PostFooter {
+  @Element() host: HTMLPostFooterElement;
+
+  /**
+   * The label to add to the footer (visually hidden).
+   */
+  @Prop() readonly label!: string;
+
+  @State() isMobile: boolean = breakpoint.get('name') === 'mobile';
+
+  connectedCallback() {
+    window.addEventListener('postBreakpoint:name', this.breakpointChange.bind(this));
+  }
+
+  private breakpointChange(e: CustomEvent) {
+    this.isMobile = e.detail === 'mobile';
+  }
+
+  render() {
```

Comment:
You could simplify the rendering logic a bit by outsourcing the individual structures to functions like

```tsx
private function renderMobileFooter() { return (<div>Accordion footer</div>); }
private function renderDesktopFooter() { return (<div>Desktop footer</div>); }

render() { return (<stuff /> { this.isMobile ? this.renderMobileFooter() : this.renderDesktopFooter() } <otherstuff/>); }
```